sonarqube-companion-rest/src/main/java/pl/consdata/ico/sqcompanion/integrations/ldap/LdapService.java

```diff
@@ -0,0 +1,45 @@
+package pl.consdata.ico.sqcompanion.integrations.ldap;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.ldap.core.AttributesMapper;
+import org.springframework.ldap.core.ContextSource;
+import org.springframework.ldap.core.LdapTemplate;
+import org.springframework.ldap.core.support.LdapContextSource;
+import org.springframework.ldap.query.ContainerCriteria;
+import org.springframework.stereotype.Service;
+import pl.consdata.ico.sqcompanion.config.AppConfig;
+import pl.consdata.ico.sqcompanion.integrations.ldap.configuration.LdapConnectionConfig;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class LdapService {
+    private final AppConfig appConfig;
+    private final LdapTemplate ldapTemplate;
+
+    public LdapTemplate getLdapTemplate() {
+        ldapTemplate.setContextSource(updateContextSource());
+        return ldapTemplate;
+    }
+
+    private ContextSource updateContextSource() {
```

Comment:
a czemu update ta metoda robi niby? :) why nie contextSource ew. ldapContextSource (chociaż to jest oczywiste w kontekście nazwy serwisu)

Reply:
rename